.github/workflows/ci.yml

```
@@ -7,33 +7,68 @@ on:
+          node-version: 22
+          cache: "pnpm"
+      - run: pnpm install
+      - run: pnpm build
 
-      - name: Build
-        run: pnpm build
+  chromatic:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
```

actionlint (1.7.12)

[error] "with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)